src/main/java/com/wrapper/Model/CareContextTable.java

```diff
@@ -0,0 +1,162 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+package com.wrapper.Model;
+
+import com.wrapper.ResponseController.LinkRecordsResponse;
+import java.util.List;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+@Document(
+		collection = "careContexts"
+)
+public class CareContextTable {
+	@Field("initRequestId")
+	@Indexed(
+			unique = true
+	)
+	public String initRequestId;
+	@Field("confirmRequestId")
+	@Indexed(
+			unique = true
+	)
```

Comment:
I am hoping that indentation and other formatting issues will be fixed once you implement spotless plugin